llvm/test/Transforms/Scalarizer/deinterleave2.ll

```diff
@@ -0,0 +1,17 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
Note: This is a replacement for the [llvm/test/Transforms/Scalarizer/sincos.ll](https://github.com/llvm/llvm-project/pull/128035/files#diff-74387c3404c6b81f15a48c66fd1b9b0d3af5e588258ca6e52b69515e3d4bce12) test (which tests the same thing).